BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -41,11 +61,16 @@ class TippingViewController: UIViewController, UIViewControllerTransitioningDele
     
     tippingView.overviewView.dismissButton.addTarget(self, action: #selector(tappedDismissButton), for: .touchUpInside)
     tippingView.optionSelectionView.sendTipButton.addTarget(self, action: #selector(tappedSendTip), for: .touchUpInside)
-    tippingView.optionSelectionView.optionChanged = { [unowned self] option in
+    tippingView.optionSelectionView.optionChanged = { [weak self] option in
+      guard let self = self else { return }
       if let balanceTotal = self.state.ledger.balance?.total {
         let hasEnoughBalanceForTip = option.value.doubleValue < balanceTotal
         self.tippingView.optionSelectionView.setEnoughFundsAvailable(hasEnoughBalanceForTip)
+      } else {
+        self.tippingView.optionSelectionView.setEnoughFundsAvailable(false)
       }
+      
+      self.tippingView.optionSelectionView.setEnoughFundsAvailable(true)
```

Comment:
This is only for testing right? so we can check out tipping flow without having BAT